qa/tasks/cephfs/filesystem.py

```diff
@@ -484,6 +491,12 @@ def __init__(self, ctx, fs_config={}, fscid=None, name=None, create=False):
         if not hasattr(self._ctx, "filesystem"):
             self._ctx.filesystem = self
 
+    def dead(self):
+        try:
+            return bool(self.get_mds_map())
```

Comment:
nit: a comment in `get_mds_map()` mentioning that it can raise `FSMissing` exception would help.

Reply:
done